packages/query/package.json

```
@@ -8,6 +8,10 @@
     "lint": "eslint \"**/*.ts*\"",
     "test": "vitest run"
   },
+  "exports": {
+    "./*": "./src/*.ts",
+    "./quieriers": "./src/queriers/*.ts"
```

```suggestion
    "./queriers": "./src/queriers/*.ts"
```

Also, isn't this case handled by `./*` above it?

---

it does seem to function without that, so removed